adapters/tappx/tappx.go

```diff
@@ -51,14 +62,31 @@ func (a *TappxAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapt
 			Message: "Error parsing bidderExt object",
 		}}
 	}
-
 	var tappxExt openrtb_ext.ExtImpTappx
 	if err := json.Unmarshal(bidderExt.Bidder, &tappxExt); err != nil {
 		return nil, []error{&errortypes.BadInput{
 			Message: "Error parsing tappxExt parameters",
 		}}
 	}
 
+	ext := Ext{
+		Bidder: Bidder{
+			Tappxkey: tappxExt.TappxKey,
+			Mktag:    tappxExt.Mktag,
+			Bcid:     tappxExt.Bcid,
+			Bcrid:    tappxExt.Bcrid,
+		},
+	}
+
+	jsonext, err := json.Marshal(ext)
+	if err == nil {
+		request.Ext = jsonext
+	} else {
+		return nil, []error{
+			fmt.Errorf("Error writing TappX request.ext."),
```

Comment:
This error doesn't contain a parameter, so please use `errors.New` rather than `fmt.Errorf`.

Reply:
Done in c140289, solving error in 56c585f.